packages/api/src/webhooks/cannon.ts

```diff
@@ -552,11 +550,15 @@ export default class WebhookCannon {
     );
     // we can't rate limit this task because it's not a user action
     await taskScheduler.createAndScheduleTask(
-      "import",
+      "upload",
       {
-        import: {
-          url: mp4RecordingUrl,
-          recordedSessionId: session.id,
+        upload: {
+          url: pathJoin(
+            this.recordingSourceUrl,
```

Comment:
So are we not getting a "recording ended" (conceptually) event from Catalyst anymore? If we do, I think it'd make more sense for this source URL to come on that payload as well. If not, this is fine.

Reply:
No, we're not calling "recording ended" from Catalyst.